pkg/operator/api/api_test.go

```diff
@@ -470,6 +523,85 @@ func TestListsNamespaced(t *testing.T) {
 		require.Error(t, err)
 		assert.Empty(t, res.GetComponents())
 	})
+	t.Run("list components for control plane", func(t *testing.T) {
+		s := runtime.NewScheme()
+		err := scheme.AddToScheme(s)
+		require.NoError(t, err)
+
+		err = componentsapi.AddToScheme(s)
+		require.NoError(t, err)
+
+		av, kind := componentsapi.SchemeGroupVersion.WithKind("Component").ToAPIVersionAndKind()
+		typeMeta := metav1.TypeMeta{
+			Kind:       kind,
+			APIVersion: av,
+		}
+		client := fake.NewClientBuilder().
+			WithScheme(s).
+			WithObjects(&componentsapi.Component{
+				TypeMeta: typeMeta,
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "obj1",
+					Namespace: "namespace-a",
+				},
+			}, &componentsapi.Component{
+				TypeMeta: typeMeta,
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "obj2",
+					Namespace: "dapr-mockns",
+				},
+				Scoped: commonapi.Scoped{
+					Scopes: []string{"dapr-placement"},
```

Comment:
What happens when I include non control plane IDs here?

Reply:
They will be ignored - Want to keep components for Apps and CP separate.